src/app/donation-start/donation-start-primary/donation-start-primary.component.ts

```diff
@@ -180,7 +177,7 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
     @Inject(ElementRef) private elRef: ElementRef,
     private formBuilder: FormBuilder,
     private identityService: IdentityService,
-    private imageService: ImageService,
+    // private imageService: ImageService,
```

Comment:
I think for future its probably better not to commit commented out code in the first place - just makes it a bit easier to follow the history if its deleted at the same time as its made unnecessary. But not worth going back and editing the commits now.

```suggestion
```